pom.xml

```diff
@@ -20,13 +20,13 @@
     <groupId>org.jboss.aerogear</groupId>
     <artifactId>unifiedpush-java-client</artifactId>
     <packaging>jar</packaging>
-    <version>1.1.0-alpha.1-SNAPSHOT</version>
+    <version>1.1.0-alpha.2-SNAPSHOT</version>
```

Comment:
ok

On Thu, Feb 5, 2015 at 5:32 PM, Sebastien Blanc <notifications@github.com>
wrote:

> In pom.xml
> <https://github.com/aerogear/aerogear-unifiedpush-java-client/pull/66#discussion_r24175785>
> :
>
> > @@ -20,13 +20,13 @@
> >      <groupId>org.jboss.aerogear</groupId>
> >      <artifactId>unifiedpush-java-client</artifactId>
> >      <packaging>jar</packaging>
> > -    <version>1.1.0-alpha.1-SNAPSHOT</version>
> > +    <version>1.1.0-alpha.2-SNAPSHOT</version>
>
> My bad, I forgot to push the commit probably and cleaned my local repos
> since then :/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-java-client/pull/66/files#r24175785>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf